roles/delay-prediction-service/templates/delay-prediction-cleanup.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
This and the timer file won't be copied to the server. You need to add them to the "Install systemd files" section.